docs/product-user-guide/src/main/asciidoc/data-objects-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='data-objects-con']
+= Data objects
+
+Data objects are the building blocks for many of the rule assets that you create, such as for guided decision tables and guided rules. Data objects are custom data types implemented as java objects in specified packages of your project. For example, you might create a Person object with data fields Name, Address, and Date of Birth to specify personal details for loan application rules. These custom data types determine what data your assets and your decision service are based on.
```

Comment:
You capitalise "Guided Decision Table" and "Guided Rule" elsewhere in the document. Did you want to be consistent here?

Reply:
Did you also want to caitalise "Java" (see "...implemented as java objects...")